n6001/hardware/ofs_n6001/build/scripts/run.sh

```diff
@@ -103,72 +108,20 @@ if [ -n "$OFS_ASP_ENV_ENABLE_ASE" ]; then
     exit $?
 fi
 
-echo "Calling compile_script"
```

Comment:
all of this adjust-plls stuff is magically done in the FIM's scripts that we bring in with pr-build-template; so we don't need it in here anymore